### ISSUE TITLE
Indent does not work

### ISSUE THREAD

**Issue by DongShelton** [palantir/tslint#3731]
### Bug Report

- __TSLint version__: 5.9.1
- __TypeScript version__: 2.7.2
- __Running TSLint via__: CLI / VSCode

#### TypeScript code being linted

```ts
const foo = () => {
console.log('hello');
}
```

with `tslint.json` configuration:

```json
{
  "defaultSeverity": "error",
  "extends": ["tslint:recommended"],
  "jsRules": {},
  "rules": {
    //...
    "indent": [true, "tabs", 2],
  },
  "rulesDirectory": []
}
```

#### Actual behavior
It doesn't fix the indent problem and also does not throw any error. 

#### Expected behavior
When run --fix or tslint auto fix in vscode, it should auto fix the indent problem.

**Comment by adidahiya** [comment]
duplicate of #2814
